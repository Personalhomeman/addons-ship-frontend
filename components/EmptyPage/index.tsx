import { Base, Flex, Text } from '@bitrise/bitkit';

import css from './style.scss';

export default () => {
  return (
    <Base className={css.emptyPage} paddingVertical="x16">
      <Flex className={css.textContainer} direction="vertical" alignChildrenVertical="middle">
        <Text align="middle" weight="bold" size="x4">
          First, you need to build an IPA/APK
        </Text>
      </Flex>
    </Base>
  );
};
