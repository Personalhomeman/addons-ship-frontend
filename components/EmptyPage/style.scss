.emptyPage {
  width: 100vw;

  .textContainer {
    background: url(/static/project-empty.svg) no-repeat center;
    height: 192px;
  }
}
