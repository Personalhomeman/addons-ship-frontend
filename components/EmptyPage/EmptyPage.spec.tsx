import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import EmptyPage from '.';

describe('EmptyPage', () => {
  it('renders correctly', () => {
    const tree = toJSON(shallow(<EmptyPage />));
    expect(tree).toMatchSnapshot();
  });
});
