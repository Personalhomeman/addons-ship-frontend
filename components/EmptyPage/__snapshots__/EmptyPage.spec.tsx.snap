// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyPage renders correctly 1`] = `
<Base
  className="emptyPage"
  paddingVertical="x16"
>
  <Flex
    alignChildrenVertical="middle"
    className="textContainer"
    direction="vertical"
  >
    <Text
      align="middle"
      size="x4"
      weight="bold"
    >
      First, you need to build an IPA/APK
    </Text>
  </Flex>
</Base>
`;
